app/gui/src/project-view/components/CircularMenu.vue

```diff
@@ -22,9 +24,17 @@ const emit = defineEmits<{
   startEditingComment: []
   openFullMenu: []
   delete: []
+  createNewNode: []
 }>()
 
+const isDropdownOpened = ref(false)
 const showColorPicker = ref(false)
+const isVisualizationEnabled = ref(props.isVisualizationEnabled)
+watch(isVisualizationEnabled, (v) => emit('update:isVisualizationEnabled', v))
```

Comment:
I think this "intermediate" ref + watch is an antipattern, and even unnecessary as I see.

Reply:
Fixed